packages/former/src/components/FormContent.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="flex flex-col items-start gap-4 m-4 justify-center">
+    <FormKit
+      type="button"
+      :label="mode === 'preview' ? 'Edit' : 'Preview'"
+      :onClick="() => (mode = mode === 'edit' ? 'preview' : 'edit')"
+    />
+    <div ref="el">
+      <FormKitSchema :schema :library />
+    </div>
+    <div class="mt-4 mx-auto">
+      <FormKit v-if="schema.length === 1" type="button" label="Add component" :onClick="addComponent" />
```

Comment:
@anbraten Why this `v-if` With this implementation you can only add one component?

Reply:
That's intended as the add button is part of each entry later on to be able to add elements between two existing ones.